packages/grid/_modules_/grid/components/panel/filterPanel/GridFilterForm.tsx

```diff
@@ -34,7 +42,12 @@ const useUtilityClasses = (ownerState: OwnerState) => {
   const { classes } = ownerState;
 
   const slots = {
-    root: ['filterForm'],
+    root: ['filterForm', 'filterPanelFilterForm'],
+    closeIcon: ['filterPanelDeleteIcon'],
```

Comment:
The owner of this slot is `GridFilterForm`. Also, some classes are not matching their slot names.

```suggestion
    deleteIcon: ['filterFormDeleteIcon'],
```

x other slots.

Reply:
I though it make more sense to refer to the FilterPanel as the owner, because if in the future we add other filtering interfaces, the name filterForm will not be enough to distinguish them.

But I assume its a good thing to be able to customize all the `'filterFormDeleteIcon'` at the same time, so yes let the naming explain how the codes work :)